README.md

```diff
@@ -0,0 +1,153 @@
+- The CID resolves to a Manifest data structure that associates code CIDs with
+  their corresponding built-in actor types.
+- The Manifest payload should be interpreted as an IPLD `Map<Cid, i32>`. Every
+  entry represents a built-in actor.
+- Manifest keys (CID) point to the Wasm bytecode of an actor as a single block.
+- Manifest values (i32) identify the actor type, to be parsed as the
+  `fvm_shared::actor::builtin::Type` enum:
+    - System = 1
+    - Init = 2
+    - Cron = 3
+    - Account = 4
+    - Power = 5
+    - Miner = 6
+    - Market = 7
+    - PaymentChannel = 8
+    - Multisig = 9
+    - Reward = 10
+    - VerifiedRegistry = 11
+
+The CARv1 is embedded as a byte slice at the root of the library, and exported
+under the `BUNDLE_CAR` public const, for easier consumption by Rust code.
+
+Precompiled actor bundles may also be provided as release binaries in this repo,
+if requested by implementors.
+
+## Instructions for client implementations
+
+### Obtaining an actors bundle
+
+There are two options:
+
+1. Building from source.
+2. Downloading the precompiled release bundle from GitHub.
+
+Instructions to build from source (option 1):
+
+1. Clone the repo.
+2. Check out the relevant branch or tag (see Versioning section below).
+3. `cargo build` from the workspace root.
+4. Copy the CARv1 file generated the location printed in this log line:
+    ```
+   warning: bundle=/path/to/repo/target/debug/build/filecoin_canonical_actors_bundle-aef13b28a60e195b/out/bundle/bundle.car
+   ```
+   All output is printed as a warning only due to limitations in the Cargo build
+   script mechanisms.
+
+Both options are compatible with automation via scripts or CI pipelines.
+
+### Integrating an actors bundle
+
+This part is implementation-specific. Options include:
+
+1. Embedding the bundle's CARv1 bytes into the distribution's binary.
+2. Downloading CARv1 files on start (with some form of checksumming for added security).
+
+### Loading and using the actors bundle with ref-fvm
+
+Once the implementation has validated the authenticity of the bundle, it is
+expected to do the following:
+
+1. Import the CARv1 into the blockstore.
+2. Retain the root CID in memory, indexed by network version.
+3. Feed the root CID to ref-fvm's Machine constructor, to tell ref-fvm which
+   CodeCID maps to which built-in actor.
+
+### Multiple network version support
+
+Because every network version may be backed by different actor code,
+implementations should be ready to load multiple actor bundles and index them
+by network version.
+
+When instantiating the ref-fvm Machine, both the network version and the
+corresponding Manifest root CID must be passed.
+
+## Versioning
+
+A fair question is how crate versioning relates to the protocol concept of
+`ActorVersion`. We adopt a policy similar to specs-actors:
+
+- Major number in crate version correlates with `ActorVersion`.
+- We generally don't use minor versions; these are always set to `0`.
```

Comment:
nit: I expect we may use patch versions, but the _network_ will use the latest patch version.

Reply:
Yes, patch versions often used late in the dev/test cycle.

Reply:
Yes, this is stated in the point below.